components/layouts/TheMenu/index.vue

```diff
@@ -49,31 +49,33 @@
             :class="{ 'menu__logo_active': isActiveHome(homeLink) }"
           />
         </AppLink>
-        <AppLink
-          v-for="link in mainLevelLinks"
-          :key="link.url"
-          class="menu__link"
-          :class="{ 'menu__link_active': isActive(link) }"
-          v-bind="link"
-        >
-          {{ link.label }}
-        </AppLink>
-      </nav>
-    </section>
-    <section
-      v-if="isCommunityActive()"
-      class="menu__second-level"
-    >
-      <nav class="menu__navigation-level">
-        <AppLink
-          v-for="link in communitySubLinks"
-          :key="link.url"
-          class="menu__link"
-          :class="{ 'menu__link_active': isActive(link) }"
-          v-bind="link"
-        >
-          {{ link.label }}
-        </AppLink>
+        <ul class="menu__navigation-level__list">
+          <li v-for="link in mainLevelLinks" :key="link.url" class="menu__navigation-level__list-item">
+            <AppLink
+              v-if="!link.sublinks"
+              class="menu__link"
+              :class="{ 'menu__link_active': isActive(link) }"
+              v-bind="link"
+            >
+              {{ link.label }}
+            </AppLink>
+            <cv-dropdown v-else class="cv-dropdown menu__navigation__dropdown" placeholder="Community">
+              <li
+                v-for="sublink in link.sublinks"
+                :key="sublink.url"
+                class="cv-dropdown-item bx--dropdown-item"
+              >
+                <AppLink
+                  class="menu__link menu__link_secondary"
+                  :class="{ 'menu__link_active': isActive(link) }"
```

Comment:
I know it is a draft, but I catch this here. I think is `isActive(sublink)`.

Reply:
great catch @korgan00 ! thank you :pray: